### PR TITLE
updated links in the standard tenant management section

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/standard-tenant/managing-permissions-bundle/global-roles.md [hunk 1]
```diff
@@ -245,13 +245,13 @@ There may be additional permissions visible depending on the features in your su
 When new features with new permissions are added to {{< product-c8y-iot >}}, these are not automatically added to existing roles. If you notice that you cannot use a new feature that was recently announced, check your permissions.
 {{< /c8y-admon-important >}}
 
-<a name="attach-global"></a>
+<a name="assign-global-roles"></a>
```
Comment (rssag) [inline content/standard-tenant/managing-permissions-bundle/global-roles.md:248]: this is now very close to the auto-generated anchor ("to-assign-global-roles")
could be removed instead

File: content/standard-tenant/managing-permissions-bundle/global-roles.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ You may also see the following legacy roles:
 |business|Can access all devices and their data but has no management permission in the tenant.
 |readers|Can read all data (including users, in contrast to "Global Readers").
 
-<a name="create-edit-roles"></a>
+<a name="add-global-role"></a>
```
Comment (rssag) [inline content/standard-tenant/managing-permissions-bundle/global-roles.md:98]: this is now very close to the auto-generated anchor ("to-add-a-global-role")
could be removed instead

**Review by rssag — APPROVED** [review]
2 suggestions regarding labels + approval

File: content/standard-tenant/managing-permissions-bundle/global-roles.md (hunk 2 above)
Comment (BeateRixen) [reply content/standard-tenant/managing-permissions-bundle/global-roles.md]: Done

File: content/standard-tenant/managing-permissions-bundle/global-roles.md (hunk 1 above)
Comment (BeateRixen) [reply content/standard-tenant/managing-permissions-bundle/global-roles.md]: Done

**Review by MWindel — APPROVED** [review]

**Pull request merged**